packages/engine/src/lib/handler/context/flow-execution-context.ts

```diff
@@ -67,7 +67,8 @@ export class FlowExecutorContext {
             return undefined
         }
         assertEqual(stepOutput.type, ActionType.LOOP_ON_ITEMS, 'stepOutput.type', 'LOOP_ON_ITEMS')
-        return stepOutput as LoopStepOutput
+        // The new LoopStepOutput is needed as casting to LoopStepOutput will just cast the data but the class methods will not be available
```

Comment:
I'm looking at this on a phone so pardon me if this is a dumb question, but why is it safe to cast to `LoopStepOutput` at all?